lms/djangoapps/ccx/views.py

```diff
@@ -234,6 +234,17 @@ def override_fields(parent, data, graded, earliest=None, ccx_ids_to_delete=None)
                 graded[block.format] = graded.get(block.format, 0) + 1
 
             children = unit.get('children', None)
+            # For a vertical override start and due dates of its all problems.
+            if unit['category'] == u'vertical':
+                for component in block.get_children():
```

Comment:
So now we're overriding due dates on all children of verticals? I think that's the safest thing to do. 

How does this interact with the performance changes edX made in https://github.com/edx/edx-platform/pull/9637 ?